File: 2-20-24/MissingNumber.py
from typing import List

class Solution:
    def missingNumber(nums: List[int]) -> int:
        expectedSum = (len(nums) * (len(nums) + 1)) / 2
        actualSum = 0
        for i in nums:
            actualSum += i
        
        return expectedSum - actualSum
    
nums = [9,6,4,2,3,5,7,0,1]
print(Solution.missingNumber(nums))

File: 2-20-24/MissingNumber.js
var missingNumber = function(nums) {
    var expectedSum = (nums.length * (nums.length + 1)) / 2;
    var actualSum = 0;
    for(var i = 0; i < nums.length; i++) {
        actualSum += nums[i];
    }

    return expectedSum - actualSum;
};

const nums = [9,6,4,2,3,5,7,0,1];
console.log(missingNumber(nums));